doc/CODING.md

```diff
@@ -51,33 +51,56 @@ intent to add a new rule here.
 
 See [DESIGN](DESIGN.md) document too, they complement each other.
 
+### Coding Style ###
 
-### C Guidelines ###
+- Limits
 
  * Functions should not exceed 100 lines.
  * Files should not exceed 1000 lines.
- * A line should not be longer than 72 characters.
- * Split up when those limits are reached.
- * Rationale: Readability with split windows.
- * The compiler shall not emit any warning (or error).
+ * A line should not be longer than 80 characters.
```

Comment:
> So ok, if you cannot live with 120, we won't use it.

Nah, I do not care that much 😀. I can totally live with 120 characters too. I still prefer 80 character (or less) though. Since you are the top contributor anyway, I think it would make sense that you have the last word on the decision. An other option would be – as KurtMi already suggested – to just let everyone vote on their favorite value.